### ISSUE TITLE
License Manager has some issues

### ISSUE THREAD

**Issue by HenryLoenwind** [kenyoni-software/godot-addons#5]
The licence.json can not be loaded (in the editor) because of `component.gd` erroring from trying to `override get_property_list()`.

```
res://addons/licenses/component.gd:78 - Parse Error: The method "get_property_list()" overrides a method from native class "Object". This won't be called by the engine and may not work as expected. (Warning treated as error.)
```

When commenting those two funcs out, `property["constructor"] = License.new` is missing, so the licences dialog won't show licenses anymore. (No, I have no idea why this worked at all and only errored when restarting the editor.)

**Comment by IceflowRE** [comment]
Can you attach an example project?
Also which Godot version do you use?
Which plugin version?

**Comment by HenryLoenwind** [comment]
v4.1.1.stable.official [bd6af8e0e]

version="1.4.1"

I think I found out why this didn't happen before. By default, Godot skips all warnings (and warnings that are promoted to errors) in addons. I recently disabled that setting because I was porting a Godot 3 addon, which led to License Manager now getting hit with that warning. And even though that override isn't respected by the engine, the code that effectively calls it probably is gdcode, not engine code. Which means without warnings it works.

I can try to make a minimal project tomorrow, but I suspect that simply toggling

![image](https://github.com/kenyoni-software/godot-addons/assets/1485873/86b25622-b5cb-4dc0-b37b-9ff7b73b3396)

is enough to trigger this.

BTW: As a quick&dirty workaround, I copied the properties tweaks into component_detail_tree's _add_item. Not nice, but works.
